scripts/i18n-build.js

```diff
@@ -0,0 +1,19 @@
+var fs = require('fs');
+var path = 'src/locales/';
+var languages = fs.readdirSync(path);
+var result = '';
+
+languages.forEach(function (lang, index) {
+	var stat = fs.statSync(path + lang);
+	if (stat.isDirectory()) {
+		var translations = fs.readFileSync(path + lang + '/main.json').toString();
+		result += '"' + lang + '": ' + translations;
+		if (index < languages.length - 1) {
+			result += ',\n';
+		}
+	}
+});
```

Comment:
with `Array#reduce` you can avoid that `result` variable.

Reply:
I think that it won't reduce number of variables used. And for me old good `forEach` is more readable.